src/Storage/Controllers/InstancesController.cs

```diff
@@ -185,7 +185,21 @@ public class InstancesController : ControllerBase
 
                 (string person, string orgNo) = InstanceHelper.SeparatePersonAndOrgNo(instanceOwnerIdType, instanceOwnerIdValue);
 
-                instanceOwnerPartyId = await _registerService.PartyLookup(person, orgNo);
+                try
+                {
```

Comment:
Should we make a distinction between PlatformException because org / person was not found (404 response) and PlatformException because bridge is unavailable or timeout for instance?

Reply:
With current state of code, what happens if it is not a PartyNotFoundException? What would be our preferred behavior here ?